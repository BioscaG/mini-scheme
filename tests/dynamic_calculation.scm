(define (sumar-dos-valors)
  (display "Introdueix el primer valor: ")
  (let ((x (read)))
    (display "Introdueix el segon valor: ")
    (let ((y (read)))
      (display "La suma és: ")
      (display (+ x y))
      (newline)
      (display "El producte és: ")
      (display (* x y))
      (newline)
      (display "El quocient és: ")
      (if (= y 0)
          (display "Indefinit (divisió per zero)")
          (display (/ x y)))
      (newline))))

(define (main)
  (sumar-dos-valors))