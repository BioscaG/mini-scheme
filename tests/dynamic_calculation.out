Introdueix el primer valor: Introdueix el segon valor: La suma és: 12
El producte és: 32
El quocient és: 2
